contrib/go-chi/chi.v5/chi.go

```diff
@@ -46,7 +47,7 @@ func Middleware(opts ...Option) func(next http.Handler) http.Handler {
 				next.ServeHTTP(w, r)
 				return
 			}
-			opts := spanOpts
```

Comment:
Line 40/41 above is  still an issue, overwriting `spanOpts`.

Reply:
Discussed offline, this is fine.